pkg/fission-cli/util/util.go

```diff
@@ -372,6 +372,20 @@ func UpdateMapFromStringSlice(dataMap *map[string]string, params []string) bool
 	return updated
 }
 
+// UpdateEnvVarFromStringSlice parses key, val from "key=val" string array and updates passed []v1.EnvVar
+func UpdateEnvVarFromStringSlice(envVar []v1.EnvVar, params []string) bool {
+	updated := false
+	for idx, m := range params {
+		keyValue := strings.SplitN(m, "=", 2)
+		if len(keyValue) == 2 {
+			envVar[idx].Name = keyValue[0]
+			envVar[idx].Value = keyValue[1]
+			updated = true
```

Comment:
If we change function prototype this should not be true.
Essentially we should make this function as much as pure as possible. Avoid side effects.